.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build and Test
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-${{ github.job }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ github.job }}-go-
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.18.2'
```

Comment:
Can an [env var](https://docs.github.com/en/actions/learn-github-actions/environment-variables) be used so we only have to update the go version in 1 place?

Reply:
:+1: done